.bazelrc

```diff
@@ -12,7 +12,7 @@ common:test --test_env=GTEST_INSTALL_FAILURE_SIGNAL_HANDLER=1
 ##
 ##  Custom toolchain.
 ##
-build --crosstool_top=@llvm_toolchain_12_0_0//:toolchain --copt=-D_LIBCPP_ENABLE_NODISCARD
+build --incompatible_enable_cc_toolchain_resolution --copt=-D_LIBCPP_ENABLE_NODISCARD
```

Comment:
I have a feeling you're more familiar with bazel toolchains than i ever have been at this point 😅 do you mind adding a comment here explaining what this bazel flag does for us? doesn't have to be long.

Reply:
Done!